src/server/common/constants/file-status.js

```diff
@@ -0,0 +1,8 @@
+const fileStatus = {
+  pending: 'pending',
+  infected: 'infected',
+  clean: 'clean',
+  complete: 'complete'
```

Comment:
I don't think `complete` makes much sense. You would have to now look for clean or complete for the success case. I would replace this with 'rejected'.

This will probably require further discussion

Reply:
Have a look over the logic in https://github.com/DEFRA/cdp-uploader/pull/69/files#diff-69f8304be64447d9ba6b317d2c6ea04778c2c0c442e456ea4d558f014b124058R59-R99 

Lets walk through it - much easier.

Reply:
essentially all the UI does now is check for

- `numberOfRejectedFiles > 0`
- `hasError === true`
- and then uses `errorMessage`